File: src/test/MainTest.java
package test;

import javamessagingnetbeans.Manager;

public class MainTest
{

    /**
     * Main test.
     *
     * @param args - program arguments.
     */
    public static void main(String[] args)
    {
        Manager manager = new Manager();

        manager.traceOn();
        manager.secretaryOn();

        manager.createEmployee(EmployeeTest.class, "EmployeeTest").start();
    }
}

File: src/test/MainTest.kt
package test

import javaasync.Manager
import neuralnetwork.services.NeuralNetwork

/**
 * Main test.
 */
object MainTest {

    /**
     * Main test.
     *
     * @param args - program arguments.
     */
    @JvmStatic
    fun main(args: Array<String>) {
        val manager = Manager()

        manager.traceOn()
        manager.secretaryOn()

        //manager.createService(AsyncServiceTest::class).start()
        manager.createService(NeuralNetwork::class).start()
    }
}
